clients/githubrepo/branches.go

```diff
@@ -476,23 +493,34 @@ nextRule:
 	return ret, nil
 }
 
+func initializedBoolRef(value bool) *bool {
+	v := new(bool)
+	*v = value
+	return v
+}
+
 func applyRepoRules(branchRef *clients.BranchRef, rules []*repoRuleSet) {
-	falseVal := false
-	trueVal := true
 	for _, r := range rules {
-		adminEnforced := len(r.BypassActors.Nodes) == 0
+		// Init values of base checkbox as if they're unchecked
 		translated := clients.BranchProtectionRule{
-			EnforceAdmins: &adminEnforced,
+			AllowDeletions:       initializedBoolRef(true),
+			AllowForcePushes:     initializedBoolRef(true),
+			RequireLinearHistory: initializedBoolRef(false),
 		}
 
+		translated.EnforceAdmins = initializedBoolRef(len(r.BypassActors.Nodes) == 0)
+
+		// Instatiate this struct as it will never be null on Repo Rules. We'll always know if PRs are required or not
+		translated.RequiredPullRequestReviews = new(clients.PullRequestReviewRule)
```

Comment:
This was also caused by our miscommunication. By the conventions I had thought, we couldn't move it to `translatePullRequestRepoRule` because this function would be called only if they required PRs. But we would need to initiate the struct anyway, because for me having the struct not nil would only mean "we have the data"

Reply:
Fixed on 2efeee6512603ac48db78db2d2551c9b65d59462

Reply:
Fixed on 2efeee6512603ac48db78db2d2551c9b65d59462

Reply:
Fixed on 2efeee6512603ac48db78db2d2551c9b65d59462

Reply:
Fixed on 2efeee6512603ac48db78db2d2551c9b65d59462

Reply:
Fixed on 2efeee6512603ac48db78db2d2551c9b65d59462